### DISCUSSION TITLE
Problem with fonts in Qfield

### DISCUSSION THREAD

**Discussion by komakeks** [opengisch/QField#5431] [Q&A]
Hello,

I tried to copy fonts to a QGIS-project but it does not work on my mobile devices. I created a folder named 'fonts' in the directory of the QGIS-project. No effect. Copying the fonts to the Android/data-folder on my mobile devices is also not working. The fonts are in ttf-format. So I changed the fonts in my QGIS-project to Roboto but this is also not working, I thought that Roboto is the standard font type on Android devices?

Can anyone help me?

**Comment by nirvn** [comment]
@komakeks , can you share your project (or at least a reduced sample project with fonts)? More details on your Android device would help too.

**Reply by komakeks** [reply]
Thanks. First device is a Galaxy S10 with Android 11, second device is a Galaxy Tab Active3 with Android 14.

**Reply by nirvn** [reply]
@komakeks , I don't see your fonts folder in there. Please attach the project you're trying to view _with the fonts_ in-place.

**Reply by komakeks** [reply]
Done.

**Reply by nirvn** [reply]
@komakeks , hm, I think this problem might linger until QGIS is built against Qt6. I did a bit of debugging, and long story short here is that the font loaded on Android has a different family name than the one in QGIS.

You might be able to 'fix it' by having a static data-defined property using the Android-loaded font family name: 'Open Sans Condensed' (vs. 'Open Sans Condensed ExtraBold' on QGIS).

**Comment by komakeks** [comment]
@nirvn Thank you very much! Static property works for me in this case. I will try other fonts, but generally this isn't so easy as told in the Qfield-manual...
